packages/documentation/src/content/docs/integration/requirements/idp.mdx

```diff
@@ -12,179 +12,123 @@ import {
-| `INTERACTION_EXPIRY_SECONDS`   | `auth.interactionExpirySeconds`    | `600`   | The time in seconds for which a user can interact with a grant request                                                                                                                                                                                                                        | Y        |
-| `INTERACTION_PORT`             | `auth.port.interaction`            | `3009`  | The port number for the [interaction endpoints](#interaction-endpoints)                                                                                                                                                                                                                       | Y        |
-| `INTROSPECTION_PORT`           | `auth.port.introspection`          | `3007`  | The port number of your Open Payments authorization token introspection server                                                                                                                                                                                                                | Y        |
-| `LIST_ALL_ACCESS_INTERACTION`  | N/A                                | `true`  | Specifies whether grant requests including a `list-all` action should require interaction. In these requests, the client asks to list resources that they themselves did not create.                                                                                                          | Y        |
+| Variable | Helm value name | Default | Description |
+| -------- | --------------- | ------- | ----------- |
+| `IDENTITY_SERVER_URL` | `auth.identityServer.domain` | N/A | Your IdP server's URL where your authorization server will direct clients to so end-users can complete the interaction and authorize a grant. | Y |
+| `IDENTITY_SERVER_SECRET` | `auth.identityServer.secret` | N/A | A shared secret between your authorization and IdP servers that your authorization server will use to secure its IdP-related endpoints.<br/>When your IdP sends requests to your authorization server, your IdP must provide the secret via an [`x-idp-secret`](#x-idp-secret-header) header. | Y |
+| `INCOMING_PAYMENT_INTERACTION` | `auth.interaction.incomingPayment` | `false` | Indicates whether incoming payments grant requests are interactive. | Y |
+| `INTERACTION_EXPIRY_SECONDS` | `auth.interactionExpirySeconds` | `600` | The time in seconds for which a user can interact with a grant request | Y |
+| `INTERACTION_PORT` | `auth.port.interaction` | `3009` | The port number for the [interaction endpoints](#interaction-endpoints) | Y |
+| `INTROSPECTION_PORT` | `auth.port.introspection` | `3007` | The port number of your Open Payments authorization token introspection server | Y |
```

Comment:
I think we don't need to include the `INTROSPECTION_PORT` here - since it's not IdP specific